Public/Src/Cache/ContentStore/Distributed/Stores/DistributedContentStoreSettings.cs

```diff
@@ -104,10 +104,6 @@ public int AssumeAvailableReplicaCount
         /// </summary>
         public bool UseTrustedHash { get; set; } = false;

```

Comment:
>UseTrustedHash [](start = 20, length = 14)

I find it odd that we can't do the same with TrustedHash yet. It predates the empty file hash optimization by a good bit.